python/dgl/data/wikics.py

```diff
@@ -0,0 +1,147 @@
+"""Wiki-CS Dataset"""
+import itertools
+import os
+import json
+import numpy as np
+from .. import backend as F
+from ..convert import graph
+from .dgl_dataset import DGLBuiltinDataset
+from ..transforms import to_bidirected, reorder_graph
+from .utils import generate_mask_tensor, load_graphs, save_graphs, _get_dgl_url
+
+
+class WikiCSDataset(DGLBuiltinDataset):
+    r"""Wiki-CS is a Wikipedia-based dataset for node classification.
```

Comment:
Add "from `Wiki-CS: A Wikipedia-Based Benchmark for Graph Neural Networks <https://arxiv.org/abs/2109.01116>`_"